style.css

```diff
@@ -0,0 +1,249 @@
+body {
+  background: #739f25;
+  background-color: snow;
+  font-family: 'Playfair Display', serif;
+  color: #739f25;
+  font-weight: bolder;
+}
+
+a {
+  color: #739f25;
+}
+
+a:link {
+  text-decoration: none;
+}
+
+a:visited {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+h4 {
+  font-size: 30px;
+  padding: 0 16px;
+  margin: 0;
+}
+
+p {
+  font-size: 30px;
+}
+
+.avocado {
+  font-size: 60px;
+}
+
+
+/*** HEADER ***/
+
+header {
+  margin: 0;
+  background: snow;
+  height: 150px;
+  display: flex;
+  align-items: center;
+  padding: 0 16px;
+}
+
+header img {
+  width: 100px;
+}
+
+.hamburger {
+  display: flex;
+  width: 30px;
+  height: 30px;
+  flex-direction: column;
+  justify-content: space-between;
+  border-radius: 5px;
+  margin-right: 24px;
+  position: absolute;
+  right: 0;
+
+}
+
+.hamburger div {
+  width: 30px;
+  height: 5px;
+  background: black;
+}
+
+.avocado {
+  display: none;
+}
```

Comment:
Seems like you are styling `.avocado` in multiple places. Best practice would be to have everything in one place - it will be easier to find once you want to add/change something